pkg/dockerregistry/server/signature.go

```diff
@@ -0,0 +1,87 @@
+package server
```

Comment:
Let's rename the file to `signaturedispatcher.go`. Or it can live in its own directory e.g. `pkg/dockerregistry/server/routes/signatures.go`. The `server` directory is getting too large.